plugin-server/src/main/services/http-server.ts

```diff
@@ -1,9 +1,9 @@
-import Piscina from '@posthog/piscina'
 import { createServer, IncomingMessage, Server, ServerResponse } from 'http'
 import { IngestionConsumer } from 'main/ingestion-queues/kafka-queue'
 import * as prometheus from 'prom-client'
 
 import { status } from '../../utils/status'
+import Piscina from '../../worker/piscina'
```

Comment:
Need to patch `collectMetrics` to remove the `if (piscina) {` branch, as the `piscina` param will not be empty.

Otherwise, we'll probably double all the counters (main + piscina==main)